bin/action_runner.py

```diff
@@ -5,79 +5,96 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+import argparse
 import contextlib
 import logging
 import os
 import subprocess
 import sys
 
-from tron import yaml
-
+import yaml
```

Comment:
Are you sure you want to do this? Before we had guarantees that we were using the C extensions and stuff

Reply:
eh, this was kind of a convenience thing so that we don't have to have the whole tron package on the batch machine. I can put it back.